stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/User.kt

```diff
@@ -1,8 +1,14 @@
 package io.getstream.chat.android.ui.utils.extensions
 
+import android.content.Context
 import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.client.models.name
 import io.getstream.chat.android.livedata.ChatDomain
+import io.getstream.chat.android.ui.R
 
-internal fun List<User>.withoutCurrentUser() = this.filter { !it.isCurrentUser() }
+internal fun List<User>.withoutCurrentUser(): List<User> = this.filter { !it.isCurrentUser() }
```

Comment:
Love these extensions, one nitpick: it would be good to not go through the `ChatDomain` inside the loop here on every iteration, but fetch the current user just a single time outside the loop.

Reply:
Agreed!